src/lib/components/Model/Model.tsx

```diff
@@ -0,0 +1,80 @@
+import { ModelDescription } from "./ModelProps/ModelDescription";
+import { Dialog } from "../../shared/components/Dialog/Dialog";
+import { Drawer } from "../../shared/components/Drawer/Drawer";
+import { Button, ButtonVariants } from "../Button/Button";
+import { ModelContent } from "./ModelProps/ModelContent";
+import { ModelFooter } from "./ModelProps/ModelFooter";
+import { ModelHeader } from "./ModelProps/ModelHeader";
+import { ModelTitle } from "./ModelProps/ModelTitle";
+import { useModelDevice } from "./useModelDevice";
+import { useModelContext } from "./ModelContext";
+import { ModelProvider } from "./ModelProvider";
+import React, { useState } from "react";
+import { ModelProps } from "./DTOs";
+
+const Model = (props: ModelProps) => {
+  const { title, description, content, submitLabel, cancelLabel, onSubmit, onCancel } = props;
+  const isDesktop = useModelDevice("(min-width: 768px)");
+  const [isModelOpen, setIsModelOpen] = useState(true);
+
+  if (isDesktop) {
+    return (
+      <Dialog open={isModelOpen} onOpenChange={setIsModelOpen}>
+        <ModelContent>
+          <ModelHeader>
+            <ModelTitle>{title}</ModelTitle>
+            <ModelDescription>{description}</ModelDescription>
+          </ModelHeader>
+          {content}
+          <Button
+            label={submitLabel || "Save changes"}
+            variant={ButtonVariants.SUCCESS}
+            onClick={onSubmit}
+          />
```

Comment:
move this into footer section

Reply:
Submit and Cancel should be in the same row
<img width="430" alt="Screenshot 2024-03-13 at 12 58 46 PM" src="https://github.com/bmesol/bmes-components/assets/91328344/db06b7fb-747c-4b92-85a9-1198abed3bcf">